aries_vcx/src/utils/devsetup.rs

```diff
@@ -57,26 +55,29 @@ pub struct SetupWalletPoolAgency {
     pub agency_client: AgencyClient,
     pub institution_did: String,
     pub wallet_handle: WalletHandle,
+    pub pool_handle: PoolHandle
 }
 
 pub struct SetupWalletPool {
     pub institution_did: String,
     pub wallet_handle: WalletHandle,
+    pub pool_handle: PoolHandle
 }
 
 pub struct SetupInstitutionWallet {
     pub institution_did: String,
     pub wallet_handle: WalletHandle,
 }
 
-pub struct SetupLibraryAgencyV2;
+pub struct SetupLibraryAgencyV2 {
```

Comment:
Not directly in scope but seems reasonably small to make this suggestion: can we rename this to `SetupPool`. Also possibly logs where `"SetupLibraryAgencyV2"` is mentioned